app/views/training_modules/index.html.slim

```diff
@@ -1,13 +1,42 @@
-h1 Training Modules
+/ https://eye-recovery.herokuapp.com/sprint-6/brain-development/course-overview
 
```

Comment:
what is this for?

Reply:
Just waiting for the Content deployment to realise it did work